src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -389,7 +402,7 @@ internal void InstantiateChildComponentOnFrame(ref RenderTreeFrame frame, int pa
             frame.ComponentIdField = newComponentState.ComponentId;
         }
 
-        internal void AddToPendingTasks(Task task)
+        internal void AddToPendingTasks(Task task, ComponentState? owningComponentState)
```

Comment:
I think we don't need this change?

The only place where `AddToPendingTasks` is called passes null as the owning component, so we are going to be passing null to handleexceptionviaerrorboundary.

Am I missing something here?